docs/low-code-jobs/databricks-jobs.md

```diff
@@ -3,23 +3,165 @@ sidebar_position: 1
+<iframe src="https://user-images.githubusercontent.com/103921419/174399585-40067429-953e-4157-a5db-d80e25713d24.mp4" title="Avro Source" allow="autoplay;fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+
+When using Databricks Jobs there are two Gem types available:
+
+#### Pipeline Gem
+
+The Pipeline Gem triggers a Spark Pipeline developed in Prophecy.
+
+![Pipeline Component](img/databricks-jobs-pipeline-config.png)
+
+Settings for Pipeline component can be inherited from overall job configuration or can be set inside the component itself.
+
+#### Script Gem
+
+Script Gem can be used to write any ad-hoc code.
+
+![Script Component](img/databricks-jobs-script-config.png)
+
+Settings for script component can be inherited from overall job configuration or can be set inside the component itself.
+
+### Visual == Code
+
+The visual graph created on the jobs page is automatically converted to code (JSON) behind the scenes, which is passed to Databricks j
+
+![Code View](img/databricks-jobs-code-view.png)
+
 ## Configuration
 
+![Example Configuration](img/databricks-job-config-example.png)
+
+---
+
+| Field Name                | Description                                                                                                                                                                                                                                         |
+| ------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Scheduler                 | The underlying engine that's going to execute your job. Databricks is recommended.                                                                                                                                                                  |
+| Fabric                    | The [execution fabric](/concepts/fabric) to which the job is going to be deployed.                                                                                                                                                                  |
+| Cluster Size              | The [default size](/concepts/fabric#whats-in-a-fabric) of the cluster, that's going to be created for the job to run.                                                                                                                               |
+| Cluster Mode              | Can be selected as `Single` (all Gems within job run on single cluster) or `Multi` (all Gems within job run on separate new cluster)                                                                                                                |
```

Comment:
Phrasing: `Single` (all Gems within the Job re-use the same Cluster) or `Multi` (all Gems within the Job run on a separate new cluster for each Gem)